xrpl/client/websocket/websocket_client.go

```diff
@@ -69,6 +70,11 @@ func (c *WebsocketClient) SendRequest(req client.XRPLRequest) (client.XRPLRespon
 	return &res, nil
 }
 
+func (c *WebsocketClient) Submit(tx interface{}, failHard bool) (client.XRPLResponse, error) {
+	_, response, error := helpers.SubmitTransaction(tx, NewClient(c.cfg), failHard)
```

Comment:
We can get rid of the helpers package and move all the `SubmitTransaction` logic here

Reply:
Let me check.
I created the `submit` helper package so it's cleaner in `websocket_client` and `jsonrpc_client`.